packages/react/src/components/Select/index.tsx

```diff
@@ -112,6 +128,17 @@ const Select = React.forwardRef(
             </span>
           )}
         </label>
+        {description && (
+          <div id={descriptionId} className="Field__description">
+            {description}
+          </div>
+        )}
+        {error && (
+          <div id={errorId} className="Error">
```

Comment:
This should be using the existing `Field__error` class that was added for `FieldGroup` https://github.com/dequelabs/cauldron/pull/1873